incident/mysql/incident.go

```diff
@@ -43,7 +44,9 @@ func NewMySQLReporter(ctx context.Context, db *sql.DB, source string) (incident.
 
 // Log records an incident with the given details.
 func (m *mysqlReporter) Log(ctx context.Context, baseURL, summary, category, fullURL, details string) {
-	if _, err := m.stmt.ExecContext(ctx, m.source, baseURL, summary, category, fullURL, details); err != nil {
+	now := time.Now()
+	glog.Errorf("%s: %s (%s %s)\n  %s at %v", baseURL, summary, category, fullURL, details, now)
```

Comment:
The logging will have a timestamp attached automatically. Are you including this one just so that the logging has a record of the exact timestamp that was written to the database?

Reply:
Yes, that's exactly it.